.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: release
+
+on:
+  push:
+    tags:
+    - '*.*.*'
+
+env:
+  DOCKER_IMAGE: registry.scality.com/static-container-registry/static-container-registry
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildk
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: registry.scality.com
+          username: ${{ secrets.REGISTRY_LOGIN }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```

Comment:
Please use the appropriate variable name: `GITHUB_REF_NAME`
See [here](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables)

Reply:
:stuck_out_tongue:

Reply:
much simpler indeed!